Feedback requested for section on inflammation, poverty, and COVID-19 outcomes (#649)
greenelab/covid19-review: content/08.transmission.md

#### Severity of COVID-19 Following Exposure

Following exposure to SARS-CoV-2, the likelihood that an individual develops COVID-19 and the severity of the disease presentation can be influenced by a number of social factors.
As discussed above, a number of patient characteristics are associated with the likelihood of severe COVID-19 symptoms.

> In some cases, these trends run counter to those expected given rates of exposure: for example, although women are more likely to be exposed, but men are more likely to be diagnosed with, hospitalized from, or die from COVID-19 [@doi:10.1371/journal.ppat.1008570].

Reviewer: ```suggestion
In some cases, these trends run counter to those expected given rates of exposure: for example, although women are more likely to be exposed, men are more likely to be diagnosed with, hospitalized from, or die from COVID-19 [@doi:10.1371/journal.ppat.1008570].
```

> In the case of comorbid conditions and racial/ethnic demographics, however, social factors are highly likely to modulate or at least influence the apparent association between the patient trait and the increased risk from COVID-19.

Reviewer: Do you mean patient here, or can that word be removed? I feel like this is true whether or not someone is seen in a clinical setting.

Author: Ah nope! I just got cognitively anchored on "patient traits" because we had such a long discussion coming up with that term!
```suggestion
In the case of comorbid conditions and racial/ethnic demographics, however, social factors are highly likely to modulate or at least influence the apparent association between these traits and the increased risk from COVID-19.
```

In particular, the comorbidities and racial/ethnic correlates of severe COVID-19 outcomes suggest that poverty confers additional risk for COVID-19.

> In order to explore the relationship between poverty and COVID-19 outcomes, it is necessary to consider how poverty impacts biology.

Author: ```suggestion
In order to explore the relationship between poverty and COVID-19 outcomes, it is necessary to consider how poverty impacts biology.
In particular, we focus on the United States and the United Kingdom.
```

Comorbidities that increase risk for COVID-19, including obesity, type II diabetes, hypertension, and cardiovascular disease, are known to be intercorrelated [@doi:10/bpp37d].
Metabolic conditions related to heightened inflammation, like obesity, type II diabetes, and hypertension, are more strongly associated with negative COVID-19 outcomes than other comorbid conditions, such as chronic heart disease [@doi:10.2337/dbi19-0023].
As discussed above, dysregulated inflammation characteristic of cytokine release syndrome is one of the greatest concerns for COVID-19-related death.
Therefore, it is possible that chronic inflammation characteristic of these metabolic conditions predisposes patients to COVID-19-related death [@doi:10.2337/dbi19-0023].

> The association between these disease and severe COVID-19 outcomes is a concern from a health equity perspective because poverty exposes people to "obesogenic" conditions [@doi:10/gddg84] and is therefore unsurprisingly associated with higher incidence of obesity and associated disorders [@doi:10.5604/12321966.1120608].

Reviewer 2: "these disease" -> "these diseases"

Reviewer 2: I know that poverty -> predisposition to obesity is definitely true in the US, but I don't think this is true everywhere. The 2nd paper you cite talks about the US and the UK. May want to specify countries for context.

Author: Thank you @SiminaB! I'm proposing that we just add a disclaimer at the beginning of this paragraph since I think all of the sources talk about the UK and USA!

Furthermore, cell phone GPS data suggests that lower socioeconomic status may also be associated with decreased access to healthy food choices during the COVID-19 pandemic [@doi:10.1002/oby.22940; @doi:10.1002/oby.22993], suggesting that health-related risk factors for COVID-19 may be exacerbated as the pandemic continues [@doi:10.1016/j.pcad.2020.07.002].
Chronic inflammation is a known outcome of chronic stress (e.g., [@doi:10.1067/mai.2000.110163; @doi:10.1037//0278-6133.21.6.531; @doi:10.1037/a0025536; @doi:10.1111/j.1467-8721.2006.00450.x]).

> Therefore, the chronic stress of poverty is likely to influence health broadly (as summarized in @doi:10.2307/26061262) and especially during the stress of the ongoing pandemic.

Reviewer 2: Again, may want to specify that this is the case for the US. We don't want to assume that what holds in the US is true everywhere.

A preprint [@doi:10.1101/2020.04.05.20054502] provided observational evidence that geographical areas in the United States that suffer from worse air pollution by fine particulate matter have also suffered more COVID-19 deaths per capita, after adjusting for demographic covariates.
Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., [@doi:10.1073/pnas.1803222115]).
Exposure to air pollution is associated with both poverty (e.g., [@doi:10.3390/ijerph15061114]) and chronic inflammation [@doi:10.1016/j.neubiorev.2018.06.002].
Taken together, the evidence suggests that low-income workers who face greater exposure to SARS-CoV-2 due to their home or work conditions are also more likely to face environmental and social stressors associated with increased inflammation, and therefore with increased risk from COVID-19.
In this case, the relationship between structural racism and COVID-19 risk becomes obvious.

> People subjected to racism are more likely to live in poverty, be incarcerated, and face food an housing instability.

Reviewer 2: Since this section isn't exclusively on racism and is focused on severity of disease following exposure, I would change the previous 2 sentences to something like: "In particular, structural racism can play an important role on disease severity after SARS-CoV-2 exposure, due to consequences of racism which include increased likelihood of poverty, incarceration, and food and housing instability."
Wouldn't incarceration fit better under "Exposure" though?

Author: Thank you so much! I made this change. I was thinking of incarceration as traumatic, but it's probably simpler to take it out!
